lib/propshaft/load_path.rb

```diff
@@ -23,7 +23,8 @@ def assets(content_types: nil)
   def manifest
     Hash.new.tap do |manifest|
       assets.each do |asset|
-        manifest[asset.logical_path.to_s] = asset.digested_path.to_s
+        manifest[asset.logical_path.to_s] ||= asset.digested_path.to_s
+        manifest[asset.non_digested_path.to_s] ||= asset.digested_path.to_s if asset.already_digested?
```

Comment:
I'm not really a fan of this duble indexing. I think we need to come up with a better solution for this one.